schedule/models/calendars.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 import pytz
-from django.contrib.contenttypes import generic
+try:
+    from django.contrib.contenttypes.generic import GenericForeignKey
+except ImportError:
+    from django.contrib.contenttypes.fields import GenericForeignKey
```

Comment:
According to the docs, this should work for both [1.7](https://docs.djangoproject.com/en/1.7/ref/contrib/contenttypes/#generic-relations) and [1.9](https://docs.djangoproject.com/en/1.9/ref/contrib/contenttypes/#generic-relations).